components_new/asp/database/SysTables.md

```
@@ -0,0 +1,38 @@
+# Database Structure
+
+### SYSTEM TABLE
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)